src/pykeen/optimizers.py

```diff
@@ -69,12 +69,5 @@
     ),
 }
 
-
-def get_optimizer_cls(query: Union[None, str, Type[Optimizer]]) -> Type[Optimizer]:
-    """Get the optimizer class."""
-    return get_cls(
-        query,
-        base=Optimizer,
-        lookup_dict=optimizers,
-        default=Adagrad,
-    )
+optimizer_resolver = Resolver(_OPTIMIZER_LIST, base=Optimizer, default=Adagrad)
```

Comment:
Not sure whether to change it here, but I suppose Adam is a more "natural" default.

Reply:
I will change it! Do you have a reason that I could include in the docs?

Reply:
Done in f418ed60